packages/core-data/src/selectors.ts

```diff
@@ -147,9 +147,9 @@ export function getCurrentUser( state: State ): ET.User< 'edit' > {
  */
 export const getUserQueryResults = createSelector(
 	( state: State, queryID: string ): ET.User< 'edit' >[] => {
-		const queryResults = state.users.queries[ queryID ];
+		const queryResults = state.users.queries[ queryID ] ?? [];
```

Comment:
Could be undefined if we don't have such a query, therefore coalescing to an empty array. That handles the nullish scenario for `.map()` below.